### DISCUSSION TITLE
Run to definition

### DISCUSSION THREAD

**Discussion by hortiSquash** [renpy/vscode-language-renpy#224] [Q&A]
in the readme there is a GIF showing the Run to Definition (F12) as well as hovering on a function to show its documentation
neither of these are working in my VSCode
is there something im missing?

![image](https://user-images.githubusercontent.com/45213805/205976120-6b11487d-ab11-479e-98d8-220ee1217aa0.png)
i mean come on its right above

No other extensions, but was installed VSCode externally from Ren'py if that makes a difference. I did start it from the Ren'Py editor though
thanks have a good day

**Comment by LuqueDaniel** [comment]
In the current version of the extension, in order to use this feature you need to open the project folder with VSCode (at the moment it doesn't work with single files). Also, the extension uses the file `project_folder/game/saves/navigation.json`. This file is generated by Ren'Py when running the project. In future versions, this will change to not depend on that file.
